backend/pkg/modules/internal/differ/http_handler.go

```diff
@@ -0,0 +1,24 @@
+package differ
+
+import (
+	"net/http"
+
+	oapicommon "github.com/openclarity/apiclarity/api3/common"
+	"github.com/openclarity/apiclarity/backend/pkg/modules/internal/common"
+)
+
+type httpHandler struct{}
+
+func (h *httpHandler) GetVersion(w http.ResponseWriter, r *http.Request) {
+	common.HttpResponse(w, http.StatusOK, &oapicommon.ModuleVersion{
+		Version: "1",
+	})
+}
+
+func (h *httpHandler) Start(w http.ResponseWriter, r *http.Request) {
+	common.HttpResponse(w, http.StatusOK, &oapicommon.ApiResponse{Message: "Start Success"})
+}
+
+func (h *httpHandler) Stop(w http.ResponseWriter, r *http.Request) {
+	common.HttpResponse(w, http.StatusOK, &oapicommon.ApiResponse{Message: "Stop Success"})
+}
```

Comment:
what is this for?

Reply:
Placeholder for when we support trace sampling

Reply:
I thought it was implemented [here](https://github.com/openclarity/apiclarity/pull/165)

Reply:
There is no differ module there... so not possible it was implemented there

Reply:
I mean that the I/S was pushed to `devel-global-specs` so you should be able to implement it, no?

Reply:
I dont know. once I will rebase on it I will see...

Reply:
implemented